<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsRasterRenderingOptionsWidgetBase</class>
 <widget class="QWidget" name="QgsRasterRenderingOptionsWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>534</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Raster Rendering Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QgsCollapsibleGroupBox" name="groupBox_14">
     <property name="title">
      <string>Bands and Resampling</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout_35">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="lbldefaultZoomedInResampling">
           <property name="text">
            <string>Zoomed in resampling</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QgsSpinBox" name="spnRed"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbldefaultZoomedOutResampling">
           <property name="text">
            <string>Zoomed out resampling</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>Blue band</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>RGB band selection</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QgsSpinBox" name="spnBlue"/>
         </item>
         <item row="0" column="4">
          <widget class="QgsSpinBox" name="spnGreen"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lbldefaultOversampling">
           <property name="text">
            <string>Oversampling</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Green band</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="6">
          <widget class="QComboBox" name="mZoomedInResamplingComboBox"/>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Red band</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="6">
          <widget class="QComboBox" name="mZoomedOutResamplingComboBox"/>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QgsDoubleSpinBox" name="spnOversampling"/>
         </item>
         <item row="4" column="0" colspan="7">
          <widget class="QCheckBox" name="mCbEarlyResampling">
           <property name="text">
            <string>Early resampling</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>35</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_17">
     <property name="title">
      <string>Contrast Enhancement</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1,0,1,0">
      <item row="1" column="3" colspan="2">
       <widget class="QComboBox" name="cboxContrastEnhancementLimitsSingleBand"/>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="cboxContrastEnhancementAlgorithmMultiBandSingleByte"/>
      </item>
      <item row="5" column="3">
       <widget class="QgsDoubleSpinBox" name="mRasterCumulativeCutUpperDoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QLabel" name="label_35">
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="cboxContrastEnhancementAlgorithmSingleBand"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_50">
        <property name="text">
         <string>Algorithm</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="2">
       <widget class="QComboBox" name="cboxContrastEnhancementLimitsMultiBandSingleByte"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>Standard deviation multiplier</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" colspan="2">
       <widget class="QComboBox" name="cboxContrastEnhancementLimitsMultiBandMultiByte"/>
      </item>
      <item row="0" column="3" colspan="2">
       <widget class="QLabel" name="label_51">
        <property name="text">
         <string>Limits (minimum/maximum)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_34">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_36">
        <property name="text">
         <string>Cumulative pixel count cut limits</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_37">
        <property name="text">
         <string>Single band gray</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="5">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QgsDoubleSpinBox" name="mRasterCumulativeCutLowerDoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_38">
        <property name="text">
         <string>Multi band color (byte / band) </string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_39">
        <property name="text">
         <string>Multi band color (&gt; byte / band) </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="cboxContrastEnhancementAlgorithmMultiBandMultiByte"/>
      </item>
      <item row="6" column="1">
       <widget class="QgsDoubleSpinBox" name="spnThreeBandStdDev">
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qgsspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spnRed</tabstop>
  <tabstop>spnGreen</tabstop>
  <tabstop>spnBlue</tabstop>
  <tabstop>mZoomedInResamplingComboBox</tabstop>
  <tabstop>mZoomedOutResamplingComboBox</tabstop>
  <tabstop>spnOversampling</tabstop>
  <tabstop>mCbEarlyResampling</tabstop>
  <tabstop>cboxContrastEnhancementAlgorithmSingleBand</tabstop>
  <tabstop>cboxContrastEnhancementLimitsSingleBand</tabstop>
  <tabstop>cboxContrastEnhancementAlgorithmMultiBandSingleByte</tabstop>
  <tabstop>cboxContrastEnhancementLimitsMultiBandSingleByte</tabstop>
  <tabstop>cboxContrastEnhancementAlgorithmMultiBandMultiByte</tabstop>
  <tabstop>cboxContrastEnhancementLimitsMultiBandMultiByte</tabstop>
  <tabstop>mRasterCumulativeCutLowerDoubleSpinBox</tabstop>
  <tabstop>mRasterCumulativeCutUpperDoubleSpinBox</tabstop>
  <tabstop>spnThreeBandStdDev</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
